feature-libs/storefinder/styles/components/_store-finder-list.scss

```diff
@@ -104,13 +104,11 @@
   }
 
   .cx-back {
-    margin-top: 0.5rem;
-    cursor: pointer;
     font-weight: 600;
-
-    &:hover {
-      color: var(--cx-color-primary);
-    }
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    max-width: 85vw;
```

Comment:
Not sure if I'm understanding reason for this max-width property. In my opinion we should rely on text size and adapt button width to the text to avoid situation where text is truncated in case super long word in some language.

Reply:
I used max-width since the button would extend past the page if there was not a limit or fixed width on the button when there was very long text in it.